syne_tune/optimizer/schedulers/searchers/multi_fidelity_searcher.py

```diff
@@ -22,13 +22,10 @@ class IndependentMultiFidelitySearcher(BaseSearcher):
     | Proceedings of the 35th International Conference on Machine Learning
 
     :param config_space: Configuration space
+    :param searcher_cls: Searcher to sample configurations on each rung level.
```

Comment:
Note: we have both `searcher` and `searcher_cls`, we could unify in `searcher` if not now we could add a todo or create an issue.

Reply:
yeah good point, I'll open an issue for this